src/selectors/archanova.js

```diff
@@ -104,30 +109,17 @@ export const useSmartWalletStatus = (): ArchanovaWalletStatus => {
   return getArchanovaWalletStatus(accounts, archanovaWalletState);
 };
 
-// Temporary stuff to get info about smart wallet type
-// TODO: combine this with etherspot work once it's in
-export const SMART_WALLET_TYPE = {
-  ETHERSPOT: ('etherspot': 'etherspot'),
-  ARCHANOVA: ('archanova': 'archanova'),
-};
-
-export type SmartWalletType = $Values<typeof SMART_WALLET_TYPE>;
-
-export const useSmartWalletType = (): SmartWalletType => {
-  return SMART_WALLET_TYPE.ARCHANOVA;
-};
-
 export const useSupportedChains = (): Chain[] => {
```

Comment:
Maybe we should move it to `utils/chains` or something, as this is not archanova related